RedditSharp/Things/LiveUpdateEvent.cs

```diff
@@ -545,7 +539,7 @@ private void CommonInit(Reddit reddit, JToken json, IWebAgent webAgent)
             WebAgent = webAgent;
         }
 
-        private void SimpleAction(string url)
+        private new void SimpleAction(string url)
```

Comment:
the "new" behavior just makes it so that it stops complaining that it hides the parent method, because the method that it is is different from the parent method and only for that file, so there's no reason to make the method virtual and have it be overridable that way